proyectos/0.md

```diff
@@ -31,6 +31,7 @@ Modifica sólo la línea que tenga tu nick de GitHub o tus iniciales (en este
 | [CommandFTL](https://github.com/Anglepi/CommandFTL)           |  0.0.1    |
 | <!-- Enlace de P O -->                |         |
 | [macime](https://github.com/soyjorgeprg/macime)        |   0.1      |
+| [Planify](https://github.com/Palinkara/Planify.git)    |   0.0.1    |
```

Comment:
Por favor, elimina el .git. La dirección correcta no incluye eso.

Reply:
Just to make sure I won't accidentally close this PR or open a second one: If I change this specific file in my forked repository and start a new PR based on that - is this counted as a second PR?

Reply:
You can simply edit it right here. Just click on changes, go to the panel for this file, and click on the tiny pencil.

Reply:
Perfect, thank you! Now it should match.